recipes/parquet-cpp/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: parquet-cpp
+  version: "0.0"
+
+source:
+  git_url: https://github.com/apache/parquet-cpp.git
+  git_rev: master
```

Comment:
@wesm pointing a recipe to `master` will render irreproducible builds. Can you change the top of your recipe to:

```yaml
{% set version = "0.0" %}
{% set commit = "c2e5f08cbefc3326535ef0ee99812b5123754d37" %}

package:
    name: parquet-cpp
    version: {{ version }}.dev0

source:
    fn: {{ commit }}.tar.gz
    url: https://github.com/apache/parquet-cpp/archive/{{ commit }}.tar.gz
    sha256: 1de1e8c2baf086d6fdffca6300d20166b5b9dbca082a0eae30a3c5b70d3e0cf4
```

I used the latest commit there BTW.